source/connection-monitoring-and-pooling/connection-monitoring-and-pooling.rst

```diff
@@ -261,6 +261,13 @@ has the following properties:
    -  Checking in a `Connection <#connection>`_ to the Pool automatically closes the `Connection <#connection>`_
    -  Attempting to check out a `Connection <#connection>`_ from the Pool results in an Error
 
+-  **Clearable:** A Pool MUST be able to be cleared. Clearing the pool marks all pooled and checked out `Connections <#connection>`_ as stale and lazily closes them as they are checkedIn or encountered in checkOut. Additionally, all requests are evicted from the WaitQueue and return retryable errors.
+
+-  **Pausable:** A Pool MUST be able to be paused and resumed. A Pool is paused automatically when it is cleared, and it can be resumed by being marked as "ready". While the Pool is paused, it exhibits the following behaviors:
```

Comment:
Can we include the justification/motivation for the paused state in the "Design Rationale" section?